### ISSUE TITLE
Error with forwardRef

### ISSUE THREAD

**Issue by LorhanSohaky** [blitz-js/babel-plugin-superjson-next#82]
```
Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `MyComponent`.
WithSuperJSON@webpack-internal:///./node_modules/babel-plugin-superjson-next/dist/tools.js:110:26
div
div
```

**Comment by Skn0tt** [comment]
Hi Lorhan! Could you provide a reproduction case? This error alone makes it very hard for me to grasp what's going on.

**Comment by LorhanSohaky** [comment]
Here an example

https://user-images.githubusercontent.com/16273730/129623822-c82dcb64-6533-4ddd-b0f0-835cd64bb37b.mp4


[example.zip](https://github.com/blitz-js/babel-plugin-superjson-next/files/6995119/example.zip)

**Comment by LorhanSohaky** [comment]
@Skn0tt , if you remove `babel-plugin-superjson-next` and `superjson` work correctly

**Comment by Skn0tt** [comment]
Will be fixed in #88. If you need a workaround until then, try renaming your `src/pages/examples` folder to something that doesn't include `pages`.

**Comment by Skn0tt** [comment]
Published in [v0.3.1](https://github.com/blitz-js/babel-plugin-superjson-next/releases/tag/v0.3.1)

**Comment by LorhanSohaky** [comment]
@Skn0tt , thanks!!
